src/release.js

```diff
@@ -28,19 +29,14 @@ module.exports = async function ({ github, context, inputs }) {
     return
   }
 
-  let releaseMeta
+  let releaseMetadata
   try {
-    releaseMeta = JSON.parse(
-      pr.body.substring(
-        pr.body.indexOf('<release-meta>') + 14,
-        pr.body.lastIndexOf('</release-meta>')
-      )
-    )
+    releaseMetadata = parseReleaseMetadata(pr)
```

Comment:
since you're moving this logic into a function, there's no need to keep error handling here, let's just do it in the function and just let an error bubble up in case it breaks, so the conditional logic here becomes simpler